controllers/provider-gcp/pkg/apis/gcp/v1alpha1/types_infrastructure.go

```diff
@@ -109,3 +111,13 @@ type CloudNAT struct {
 	// +optional
 	MinPortsPerVM *int32 `json:"minPortsPerVM,omitempty"`
 }
+
+// FlowLog contains the configuration options for the vpc flow logs
+type FlowLog struct {
+	// AggregationInterval for collecting flow logs
+	AggregationInterval string `json:"aggregationInterval,omitempty"`
```

Comment:
Is this an optional or a required field? If optional, then it must be a `*string` and `// +optional` tag should be there. If required, then remove `omitempty`.